### DISCUSSION TITLE
How are people getting dark mode to work?

### DISCUSSION THREAD

**Discussion by kirkbushell** [tailwindlabs/tailwindcss#4767] [Help]
I have dark mode enabled in taildwind:

`darkMode: 'media',`

I then set the color based on dark mode:

`dark:bg-gray-100`

Gray is never applied. Check chrome, inspect element, no class is applied there. Chrome dark mode is enabled, as is OS dark mode. 

I also found examples online, also not working: https://play.tailwindcss.com/RpfeDooXVs

Help?

**Comment by surjithctly** [comment]
Media means, you have to enable dark mode on your system, Then it will show automatically. Check articles on how to enable dark mode on Windows, Mac, or Linux. 

To test dark mode, Open Chrome Devtools and Press `Ctrl + Shift + P` and then search for dark emulation, Click on it and you can test dark mode without enabling actually.

**Reply by kirkbushell** [reply]
It's enabled already, so that's not particularly helpful :)

**Reply by surjithctly** [reply]
Then, you should provide more details. It's hard to imagine what's going on in your system.

**Reply by kirkbushell** [reply]
I'm unsure what else you need - I've provided the information I have.

media mode for darkMode is enabled. I also found out that the chrome setting I had was breaking it - I set it simply to "enabled" (instead of default) and it works (on the link provided above). But not on my own project.

No idea what's going on.

**Reply by surjithctly** [reply]
Any public link I can check? if not, you can record a tella.tv or loom video.
